frame/bags-list/remote-tests/src/snapshot.rs

```diff
@@ -38,10 +38,6 @@ pub async fn execute<Runtime: crate::RuntimeT, Block: BlockT>(
 		}))
 		.inject_hashed_prefix(&<pallet_staking::Bonded<Runtime>>::prefix_hash())
 		.inject_hashed_prefix(&<pallet_staking::Ledger<Runtime>>::prefix_hash())
-		.inject_hashed_prefix(&<pallet_staking::Validators<Runtime>>::prefix_hash())
-		.inject_hashed_prefix(&<pallet_staking::Nominators<Runtime>>::prefix_hash())
-		.inject_hashed_key(&<pallet_staking::CounterForNominators<Runtime>>::hashed_key())
```

Comment:
Why are you removing `Validators ` as well? and why is `CounterForNominators` not being replaced? It is probably needed in this context.

Reply:
I was a bit skeptical on what to do here. I tried to replace `CounterForNominators` but I couldn't find a `hashed_key` function to use here. Also didn't find a `prefix_hash` function for the `Nominators `and `Validators`.

Reply:
Well, then they should be created. Removing them is like cutting corners here :p

Reply:
This should be fun

Reply:
should be fixed now we have the 2 key and the 2 prefix